minimal/compose/certbot/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM palobo/certbot:latest
```

Comment:
Agreed. I'll update that to tags. Just need to correct a few things on Docker hub first.